packages/core/src/server/fetch.js

```diff
@@ -1,22 +1,34 @@
 import fetch from 'cross-fetch';
+import AbortController from 'abort-controller';
```

Comment:
Do we need this? Should be native in nodejs (as of v16) and browser.

Reply:
only `vnext-api` is on 16, everything else is on 14